project-koku/koku
# COST-2107: Remove unnecessary AWS Partables.

## HOW TO TEST
1. Check out main and load the database with aws data
2. Log into the db and check to make sure the partable tables are there:
```
postgres=# select table_schema, 
       table_name 
  from information_schema.tables 
 where table_name in (
           'reporting_aws_compute_summary_by_region_p', 
           'reporting_aws_compute_summary_by_service_p', 
           'reporting_aws_storage_summary_by_region_p', 
           'reporting_aws_storage_summary_by_service_p'
       );

 table_schema |                 table_name
--------------+--------------------------------------------
 template0    | reporting_aws_compute_summary_by_region_p
 template0    | reporting_aws_compute_summary_by_service_p
 template0    | reporting_aws_storage_summary_by_region_p
 template0    | reporting_aws_storage_summary_by_service_p
 acct10001    | reporting_aws_compute_summary_by_service_p
 acct10001    | reporting_aws_compute_summary_by_region_p
 acct10001    | reporting_aws_storage_summary_by_region_p
 acct10001    | reporting_aws_storage_summary_by_service_p
(8 rows)
```
3. Check out this branch, and run the migrations
```
make run-migrations
```
4) Check the db again
```
postgres=# select table_schema,
       table_name
  from information_schema.tables
 where table_name in (
           'reporting_aws_compute_summary_by_region_p',
           'reporting_aws_compute_summary_by_service_p',
           'reporting_aws_storage_summary_by_region_p',
           'reporting_aws_storage_summary_by_service_p'
       );
 table_schema | table_name
--------------+------------
(0 rows)
```
